platform/ios/src/MGLMapView.mm

```diff
@@ -389,7 +389,9 @@ - (void)commonInit
     _accessibilityCompassFormatter.unitStyle = NSFormattingUnitStyleLong;
     self.backgroundColor = [UIColor clearColor];
     self.clipsToBounds = YES;
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000
```

Comment:
This is a bit different than checking the Xcode or compiler version. I wonder if there’s a `__has_feature()` or `__has_attribute()` flag that indicates whether the current Clang version supports [`@available` expressions](https://clang.llvm.org/docs/LanguageExtensions.html#objective-c-available).